python/py-pygraphviz/Portfile

```diff
@@ -25,7 +25,7 @@ checksums           rmd160  95e03bbc2ca3f0cac2549eeb8763ccc1ef3fa7a0 \
                     sha256  5d34274833b21d9ed69cecd1abb9c1f2e12abe1856c5197809a38e8df8229ae7 \
                     size    111080
 
-python.versions     310 311 312
+python.versions     310 311 312 313
```

Comment:
please amend the commit message for the last commit to:
"py-pygraphviz: add py313 subport"

Reply:
Done